src/frontend/sharedComponents/WifiBar.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { StyleSheet, TouchableNativeFeedback, View } from "react-native";
```

Comment:
Use export from `./Touchables` - it will make adding iOS support easier in the future (since it exports different things based on platform), and it uses the react-native-gesture-handler touchable, that has better performance, although I think the new "Pressable" in react-native might have solved this now.